common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/core/validation/InetAddressValidator.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Plugin can have in pom.xml a task that downloads the jar into folder 'lib' (registered in .gitignore) and .classpath will reference the jar. As an example, plugin org.jboss.tools.openshift.client downloads some jars in this way.